content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md

```diff
@@ -0,0 +1,194 @@
+Kubernetes infrastructure hosted on pkgs.k8s.io!**
+
+There will be a deprecation phase for the existing packages, which is currently
+being [discussed in the CRI-O community](https://github.com/cri-o/cri-o/discussions/7315).
+The new infrastructure will only support releases of CRI-O `>= v1.28.2` as well as
+the corresponding release branches.
+
+## How to use the new packages
+
+In the same way as the Kubernetes community, CRI-O provides `deb` and `rpm`
+packages as part of a dedicated subproject in OBS, called
+[isv:kubernetes:addons:cri-o](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o).
+This project acts as an umbrella and provides `stable` (for CRI-O tags) as well as
+`prerelease` (for CRI-O `releaes-1.y` and `main` branches) package builds.
+
+**Stable Releases:**
+
+- [`isv:kubernetes:addons:cri-o:stable`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable): Stable Packages
+  - [`isv:kubernetes:addons:cri-o:stable:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.29): `v1.29.z` tags
+  - [`isv:kubernetes:addons:cri-o:stable:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.28): `v1.28.z` tags
+
+**Prereleases:**
+
+- [`isv:kubernetes:addons:cri-o:prerelease`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease): Prerelease Packages
+  - [`isv:kubernetes:addons:cri-o:prerelease:main`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:main): [`main`](https://github.com/cri-o/cri-o/commits/main) branch
+  - [`isv:kubernetes:addons:cri-o:prerelease:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.29): [`release-1.29`](https://github.com/cri-o/cri-o/commits/release-1.29) branch
+  - [`isv:kubernetes:addons:cri-o:prerelease:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.28): [`release-1.28`](https://github.com/cri-o/cri-o/commits/release-1.28) branch
+
+There are no stable releases available in the v1.29 repository yet, because
+v1.29.0 will be released in December. The CRI-O community will also **not**
+support release branches older than `release-1.28`, because there have been CI
+requirements merged into `main` which could be only backported to `release-1.28`
+with appropriate efforts.
+
+For example, If an end-user would like to install the latest available version
+of the CRI-O `main` branch, then they can add the repository in the same way as
+they do for Kubernetes.
+
+### `rpm` Based Distributions
+
+For `rpm` based distributions, you can run the following commands as a `root` user
+to install CRI-O:
+
+#### Add the Kubernetes repo
+
+```bash
+cat <<EOF | tee /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://pkgs.k8s.io/core:/stable:/v1.28/rpm/
+enabled=1
+gpgcheck=1
+gpgkey=https://pkgs.k8s.io/core:/stable:/v1.28/rpm/repodata/repomd.xml.key
+EOF
+```
```

Comment:
We can potentially drop this because you can use CRI-O repos without Kubernetes repos. Instead, we can link to the official installation docs.

Reply:
We require `kubernetes-cni` in the CRI-O spec.

Reply:
Can we add this as a note? Something like: "we need the kubernetes repo because CRI-O depends on the `kubernetes-cni` package hosted in that repo"